projects/plugins/jetpack/modules/stats.php

```diff
@@ -39,23 +39,10 @@
 function stats_load() {
 	Jetpack::enable_module_configurable( __FILE__ );
 
-	// Generate the tracking code after wp() has queried for posts.
-	add_action( 'template_redirect', 'stats_template_redirect', 1 );
```

Comment:
The extracted actions will now run on `plugins_loaded`, while the ones left here on `jetpack_modules_loaded`.